falcon/response_helpers.py

```diff
@@ -60,3 +61,18 @@ def format_range(value):
             str(value[0]) + '-' +
             str(value[1]) + '/' +
             str(value[2]))
+
+
+def is_ascii_encodable(s):  # pragma: no cover
+    """ check if argument encodes to ascii without error
```

Comment:
Nitpick: No preceding whitespace, and capitalize the 'c' in 'check'. Also, include a period and remove the newline, i.e.:

```"""Check if argument encodes to ascii without error."""```